lms/djangoapps/courseware/constants.py

```diff
@@ -0,0 +1,8 @@
+"""
+Constants for courseware app.
+"""
+UNEXPECTED_ERROR_IS_ELIGIBLE = "An unexpected error occurred while fetching " \
+                               "financial assistance eligibility criteria for a course"
+UNEXPECTED_ERROR_APPLICATION_STATUS = "An unexpected error occurred while getting " \
+                                      "financial assistance application status"
+UNEXPECTED_ERROR_CREATE_APPLICATION = "An unexpected error occurred while creating financial assistance application"
```

Comment:
We can factor out `An unexpected error occurred while` string as well. Not a blocker.